packages/host/app/services/message-service.ts

```diff
@@ -1,7 +1,8 @@
 import Service, { service } from '@ember/service';
+
 import { tracked } from '@glimmer/tracking';
 
-import type LoaderService from './loader-service';
+import LoaderService from './loader-service';
```

Comment:
Seems like this should still be a type import

Reply:
Fixed.